doc/configure.md

```diff
@@ -0,0 +1,277 @@
+command_buffer_size = 16384
+worker_count = 2
+handle_process_affinity = false
+max_connections = 500
+max_buffers = 500
+buffer_size = 16384
+```
+
+### Listeners
+
+The _listener_ section describes a set of listening sockets accepting client connections.
+You can define as many listeners as you want.
+They follow the format:
+
+*General parameters:*
+
+```toml
+[[listeners]]
+# possible values are http, https or tcp
+protocol = "http"
+# listening address
+address = "127.0.0.1:8080"
+
+# specify a different IP than the one the socket sees, for logs and forwarded headers
+# public_address = "1.2.3.4:80
+
+# Configures the client socket to receive a PROXY protocol header
+# expect_proxy = false
+```
+
+#### Options specific to HTTP and HTTPS listeners
+
+```toml
+# path to custom 404 and 503 answers
+# a 404 response is sent when sozu does not know about the requested domain or path
+# a 503 response is sent if there are no backend servers available
+answer_404 = "../lib/assets/404.html"
+answer_503 = "../lib/assets/503.html"
+
+# defines the sticky session cookie's name, if `sticky_session` is activated format
+# an application. Defaults to "SOZUBALANCEID"
+sticky_name = "SOZUBALANCEID"
+```
+
+#### Options specific to HTTPS listeners
+
+```toml
+# supported TLS versions. Possible values are "SSLv2", "SSLv3", "TLSv1",
+# "TLSv1.1", "TLSv1.2", "TLSv1.3". Defaults to "TLSv1.2"
+tls_versions = ["TLSv1.2"]
+```
+
+#### Options specific to Openssl based HTTPS listeners
+
+```toml
+#option specific to Openssl based HTTPS listeners
+cipher_list = "ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256"
+```
+
+#### Options specific to Rustls based HTTPS listeners
+
+```toml
+#option specific to rustls based HTTPS listeners
+rustls_cipher_list = ["TLS13_CHACHA20_POLY1305_SHA256"]
+```
+
+### Applications
+
+You can declare the list of your _applications_ under the `[applications]` section.
+They follow the format:
+
+*Mandatories parameters:*
+
+``` toml
+[applications]
+
+[applications.NameOfYourApp]
+# possible values are http or tcp
+# https proxies will use http here
+protocol = "http"
+
+# per application load balancing algorithm. The possible values are
+# "roundrobin" and "random". Defaults to "roundrobin"
+# load_balancing_policy="roundrobin"
+
+# force application to redirect http traffic to https
+# https_redirect = true
+
+frontends = [
+  { address = "0.0.0.0:8080", hostname = "lolcatho.st" },
+  { address = "0.0.0.0:8443", hostname = "lolcatho.st", certificate = "../lib/assets/certificate.pem", key = "../lib/assets/key.pem", certificate_chain = "../lib/assets/certificate_chain.pem" }
+]
+# additional options for frontends: sticky_session (boolean)
+
+backends  = [
+  { address = "127.0.0.1:1026" }
+]
+```
+
+## Sozuctl
```

Comment:
the command socket is not specific to sozuctl: it is used by every tool commanding sozu. We should document that usage though. Sozuctl will not take the command socket directly as argument, instead it reads it from the configuration file